src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  return (
+    <>
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age ? (
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      ) : null}
+      <p className="Person__partner">{person.partnerName ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}` : `I am not married`}</p>
```

Comment:
Create a variable for husband/wife. Also, fix formatting to make your code cleaner
![image](https://github.com/mate-academy/react_person/assets/96409563/723a4501-707c-4b20-aa41-ae7a2aa1c96b)
![image](https://github.com/mate-academy/react_person/assets/96409563/554969c8-7cfc-4930-9291-b5ba28b1ee15)